packages/nextjs/src/app-beta/currentUser.ts

```diff
@@ -1,7 +1,9 @@
+import type { User } from '@clerk/backend';
```

Comment:
❓ should this be an import from the `@clerk/backed` or something from `@clerk/types` ?